### PR TITLE
Feature modals

### PR DESCRIPTION
**Pull request by afzalsayed96**
## Brief description

Added modals using CSS

## Developer Certificate of Origin

- [x] I certify that these changes according to the Developer Certificate of Origin 1.1 as described at <https://developercertificate.org/>.

## Sample pictures

image
image
image
image

## Further details

As discussed in issue #95 modals using only CSS has some drawbacks:
- Modal does not close by pressing `esc` key or clicking anywhere outside
- Pressing back button after closing the modal will make it appear again
- Parent scroll cannot be disabled which is bad for material design principles

### CODE REVIEW

**Comment by rhyneav** [comment]
Definitely see your point with the potential issues with only using CSS. Would it be worth introducing a section for components that need JavaScript, or should this project remain purely CSS?

**Comment by rhyneav** [comment]
But they look great nonetheless! <3

**Comment by TotomInc** [comment]
IMO it should stay as a pure CSS project @rhyneav

**Comment by rhyneav** [comment]
That sounds good @TotomInc. I was starting to stray from the light...

**Comment by Fraham** [comment]
Yeah i agree, if we wanted to add js elements it should be in a separate repo

**Comment by TotomInc** [comment]
@Fraham like in the react or vuejs version... 😉

File: index.html [hunk 1]
```diff
@@ -1229,7 +1230,96 @@ <h1 class="article-title"><a href="">Article Title</a></h1>
...
+  &lt;/div&gt;
+&lt;/div&gt;</code></pre>
+        </div>
+
+      <h4>Paper border</h4>
+      <div class="row flex-spaces child-borders">
+        <a href="#modal-3" class="paper-btn margin">Paper Modal!</a>
+      </div>
+      <div class="modal row flex-spaces" id="modal-3">
+        <div class="modal-body border">
```
Comment (rhyneav) [inline index.html]: I'm personally not crazy about the look of a paper-border with modals. I think modals could represent a digital sticky note which would have sharp edges. The default styles look good!

**Comment by Fraham** [comment]
Yeah

**Comment by Fraham** [comment]
@afzalsayed96 would you be comfortable adding tests that cover the new less code you have added?

**Comment by afzalsayed96** [comment]
@Fraham I don't know.. Can you guide me how to do it?
@rhyneav CSS modals work fine in in firefox mobile and web as well as edge & chrome in desktop. Mobile view in chrome is a bit disoriented as pointed in issue #103.

File: index.html (hunk 1 above)
Comment (afzalsayed96) [reply index.html]: Should we remove the 3rd modal then?

Comment (rhyneav) [reply index.html]: Please!

**Comment by rhyneav** [comment]
I think this is good to go! @Fraham are you comfortable with not having tests for this yet?

**Comment by Fraham** [comment]
Yeah i am happy without them

**Comment by afzalsayed96** [comment]
@Fraham I would love to learn about writing tests in future if you can help
me how to start with it.

**Pull request merged**